src/ParallelHashMap.h

```diff
@@ -363,8 +360,8 @@ K* FixedHashMap<K,V>::keys()
  *      key/value pairs in the table.
 */
 template <class K, class V>
-V* FixedHashMap<K,V>::values()
-{
+V* FixedHashMap<K,V>::values() {
+
   /* allocate array of values */
   V *values = new V[_N];
 
```

Comment:
Can you place the `{` on lines 371 and 374 after their corresponding statements on the previous line?